bindings/python/tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+from uuid import uuid4
+import os
+
+import pytest
+import libtorrent as lt
+
+
+@pytest.yield_fixture
+def torrent_info(tmpdir):
```

Comment:
what does this function do? it doesn't look like it's testing anything.

Reply:
https://docs.pytest.org/en/latest/fixture.html